highfidelity/hifi
# Case 21562: Audio settings screen

# TEST PLAN
- Launch interface
- start up some talking bots or go to a domain with someone
- go to the audio app in the tablet
- change the 'avatar volume' slider at the bottom
- volume of the avatars in the domain should change accordingly based on user setting
- open people app in the tablet
- master volume should not be visible
- go to the audio app in the tablet
- Click 'test your voice'
- speak into your input microphone and you should hear it in your output
- click the 'stop testing your voice', or wait 8 seconds.  Testing your voice will stop
- visuals for the settings screen should follow version 3 in the Figma document

https://highfidelity.manuscript.com/f/cases/21562/Audio-Loopback-Button-Audio-Settings-Screen
